### PR TITLE
Variables of vehicle and trailer added

### PR DESCRIPTION
**Pull request by lfc-256**
Some variables used in de dynamics model of the vehicle and trailer added.

### CODE REVIEW

**Comment by HugoLeyvaOax** [comment]
Very well.

**Review by HugoLeyvaOax — APPROVED** [review]

File: Scripts/variable_dictionary.m [hunk 1]
```diff
@@ -5,6 +5,9 @@
 %   and trailer parameters that are loaded and used in the simulation (Simulink).
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+% No. Manuever
```
Comment (jcontrolresearch) [inline Scripts/variable_dictionary.m]: %% Maneuver number 

Use %% to split code in sections. This in order to document easily in the future

File: Scripts/variable_dictionary.m [hunk 2]
```diff
@@ -17,5 +20,20 @@
 v=5.0;      %Speed  (m/s)
 Df=1;       %Angle between the front wheel and the car  (rad)
 
+%Vehicle parameters 2
```
Comment (jcontrolresearch) [inline Scripts/variable_dictionary.m]: Why is call Vehicle parameter 2?

**Review by jcontrolresearch — CHANGES_REQUESTED** [review]
Please review comments and commit fixes

**Comment by jcontrolresearch** [comment]
@JControlReserch please update types of trailer ASAP

**Comment by jcontrolresearch** [comment]
I take a look into trailers types but actually is assigned to you guys @lfc-256, @raul-rd, @SaraML on issue #13. Please take a look.

Anyway I found this table which is pretty useful at start point 
image

Please use as starting point.
Consider to automate master script to analyse the control algorithm in another point of view. We will discusses this on next stand up.

Please look in this [link ](https://www.mariotrailers.com.au/blog/types-of-car-trailers/)as reference.

**Review by jcontrolresearch — APPROVED** [review]
We follow matrix issue on assigned pull request, now I merge dict file to master

**Pull request merged**